src/main/java/org/opensearch/search/relevance/transformer/personalizeintelligentranking/PersonalizeRankingResponseProcessor.java

```diff
@@ -150,7 +153,7 @@ public Factory(PersonalizeClientSettings settings) {
         }
 
         @Override
-        public PersonalizeRankingResponseProcessor create(Map<String, Processor.Factory<SearchResponseProcessor>> processorFactories, String tag, String description, Map<String, Object> config) throws Exception {
+        public PersonalizeRankingResponseProcessor create(Map<String, Processor.Factory<SearchResponseProcessor>> processorFactories, String tag, String description, boolean ignoreFailure, Map<String, Object> config, PipelineContext pipelineContext) throws Exception {
```

Comment:
Is ``pipelineContext`` expected to be used somewhere?

Reply:
pipelineContext is Contextual information about the enclosing pipeline and it's used in the metrics [PipelineWithMetrics](https://github.com/opensearch-project/OpenSearch/blob/051550112c7ae7ae1e81f9da3eff10c73407cf38/server/src/main/java/org/opensearch/search/pipeline/SearchPipelineService.java#L193) 
But PersonalizeRankingResponseProcessor doesn't change the pipelineContext, should be good here

Reply:
Sounds good. thanks for the context.